api/Services/CarParkService.js

```diff
@@ -62,5 +69,13 @@ const getAvailableCarParks = async (carParkCollection, duration) => {
 
     return carParks
 }
+
+let postBooking = async (bookingCollection, newBooking) => {
+    const result = await bookingCollection.insertOne(newBooking)
+    return result;
+}
```

Comment:
Would suggest putting anything booking related into a BookingsService.js file instead of putting it in here